hw/top_earlgrey/ip_autogen/alert_handler/fpv/tb/esc_after_load.tcl

```diff
@@ -0,0 +1,10 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# The UpCntIncrStable_A and DnCntIncrStable_A assertions are vacuous
+# in this test because we can never get to the situation where the
+# counter would saturate: it only be set to 1 and incremented, but it
+# is reasonably wide (EscCntDw=32).
```

Comment:
Hmm, I guess so. Another way to do it would be to have a magic parameter that pulls in a magic value to force things (which is the approach we take for the SecCmFpv tests). I'll file an issue tracking the fact that we might want to do this in future. (It's pretty low priority, given that `prim_count` has this assertion checked exhaustively for other instantiations, where the `set_i` signal can jump things).

Reply:
Issue raised as #22824.